pkg/minikube/kubeconfig/config_test.go

```diff
@@ -1,72 +0,0 @@
-/*
```

Comment:
Since, we are removing unit test to verify kubeconfig entries. 
It would be good to verify it as part of integration test.

Reply:
@budhrg as part of integration test it's already verified because this is the step which part of `postclusterup` and if there is any issue with kubeconfig file then start minishift fails. Even with the deploying and app or getting the routes of applications all depend on this file.  I am not sure what else you want to add as part of test.

Reply:
As this time, I am approving as during application deployment, it get tested.

I was thinking of testing the content of minishift_kubeconfig file after start succeed.

Reply:
@budhrg Testing minishift_kubeconfig context doesn't make sense because it changes with each VM creation, if you think only to test structure that again not something useful as a test, but as I told if our generated config have any issue then it will fail in every step (be it starting minishift, getting the service/route for service ... etc.)